dags/tmf/bikeshare/trips.py

```diff
@@ -0,0 +1,197 @@
+"""
+Download Los Angeles Metro Bikeshare trip data from Tableau,
+upload to Postgres and S3.
+"""
+import io
+import logging
+import os
+from datetime import datetime, timedelta
+
+import pandas
+import sqlalchemy
+import tableauserverclient
+from airflow import DAG
+from airflow.hooks.postgres_hook import PostgresHook
+from airflow.hooks.S3_hook import S3Hook
+from airflow.models import Variable
+from airflow.operators.python_operator import PythonOperator
+
+S3_ID = "s3_conn"
+POSTGRES_ID = "postgres_default"
+SCHEMA = "tmf"
```

Comment:
do we want yet another schema for the TMF project? wondering if we should combine all transportation data?

Reply:
Yeah, you're probably right. These tables are general-enough interest to be independent of a TMF schema.